### DISCUSSION TITLE
i used reactjs umi app, when i run git commit, there is an error  Unexpected unknown at-rule "@tailwind"

### DISCUSSION THREAD

**Discussion by Cooper1886** [tailwindlabs/tailwindcss#11309] [Help]
this issue has prevented my submission, how can I handle it

**Comment by Rafaelaxo** [comment]
Looking for the same issue. Bumped into your thread. Thanks for creating it. Looking forward for solution. [MyBKExperience Survey](https://www.mybkexperience.one/)

**Reply by Cooper1886** [reply]
![image](https://github.com/tailwindlabs/tailwindcss/assets/107901138/b2970684-1af7-4302-b832-b84283e41964)
handle as shown in the photo, it works. 
but i dont know how to handle this problem while maintaining the original coding style
